config/dev-amo.js

```diff
@@ -31,4 +31,6 @@ module.exports = {
   },
 
   extensionWorkshopUrl: 'https://extensionworkshop-dev.allizom.org',
+
+  enableFeatureUseAdzerkForSponsoredShelf: true,
```

Comment:
I'm not entirely sure about this. If we want QA to be able to test *this* feature, then we need this, but if we make any changes to the existing sponsored shelf, and we want those tested, then this would preclude that. I suppose we just keep that in mind, and if there is a patch in the future to the current version of the sponsored shelf, then we could flip this flag in that patch.

Reply:
This also means that QA cannot check for regressions that might have been introduced by this patch in the current behaviour of the sponsored shelf, which is also less than idea. I'm not sure how to best deal with this, as flipping the flag requires a commit to land.

Reply:
QA can still use stage to verify issues relate to the current sponsored shelf. It's not ideal but better than nothing.